ci/autoresearch/phases.py

```
@@ -1621,6 +1627,48 @@ async def _run_lpc_pin_toggle_rx_tests(ctx: RunContext) -> int:
+        "uv",
+        "run",
+        "python",
+        "ci/autoresearch/test_lpc_ws2812_loopback.py",
+        "--port",
+        upload_port,
+        "--tx-pin",
+        str(tx_pin),
+        "--rx-pin",
+        str(rx_pin),
+    ]
+    result = subprocess.run(cmd)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)